transferUserPasswordsToHashes.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const bcrypt = require('bcrypt');
+
+users = require('./users.json');
+Object.keys(users).forEach(function(user) {
+    users[user] = bcrypt.hashSync(users[user], 8);
+});
+
```

Comment:
maybe we can rewrite this file to create a cli to add a user

Reply:
I have created another file which can read a username and password from CLI, hashes the password and writes it to the users.json